### PR TITLE
[#168821506] Add ValidationToken entity

### PR DESCRIPTION
**Pull request by francescopersico**
I am splitting #31 in 2 parts.
Second part here.

### CODE REVIEW

**Comment by francescopersico** [comment]
Changes already approved in #31 so i am going to merge.

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#168821506](https://www.pivotaltracker.com/story/show/168821506): Quando nel profilo viene cambiata l'email deve partire il processo di verifica

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0575d64e767d271b8bff294e6cce1f8fb4126610
</p>

File: src/entities/validation_token.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { UTCISODateFromString } from "italia-ts-commons/lib/dates";
+import { EmailAddress } from "../../generated/definitions/EmailAddress";
+import { FiscalCode } from "../../generated/definitions/FiscalCode";
+import { TableEntity } from "../utils/azure_storage";
+
+export const VALIDATION_TOKEN_TABLE_NAME = "ValidationTokens";
+
+/**
+ * Used to describe properties of the ValidationToken storage table.
+ * By convention storage table properties uses PascalCase.
```
Comment (gunzip) [inline src/entities/validation_token.ts]:
```suggestion
 * By convention storage table properties use PascalCase.
```

File: src/utils/azure_storage.ts [hunk 2]
```diff
@@ -222,3 +226,114 @@ export async function getBlobAsObject<A, O, I>(
...
+export const TableEntity = t.interface({
+  PartitionKey: t.string,
+  RowKey: t.string
+});
+export type ITableEntity = t.TypeOf<typeof TableEntity>;
+
+// Code used by tableService when an entity is not found
+export const ResourceNotFoundCode = "ResourceNotFound";
+
+// Describe a entity returned by the retrieveEntity function
```
Comment (gunzip) [inline src/utils/azure_storage.ts]:
```suggestion
// Describe an entity returned by the retrieveEntity function
```

File: src/utils/azure_storage.ts [hunk 3]
```diff
@@ -222,3 +226,114 @@ export async function getBlobAsObject<A, O, I>(
...
+ * `"Key": {"&": DataType, "_": Value}`
+ *
+ * this function converts to an object containing only the values like:
+ *
+ * `"Key": Value`
+ *
+ * @param entityResult the object returned by TableService retrieveEntity function
+ */
+export function getValueOnlyEntityResolver(
+  // We use Object becuase it is required by the azure-storage TableEntityRequestOptions type
```
Comment (gunzip) [inline src/utils/azure_storage.ts]:
```suggestion
  // We use 'Object' as it is required by the azure-storage TableEntityRequestOptions type
```

**Pull request merged**